ote_sdk/ote_sdk/tests/serialization/test_label_mapper.py

```diff
@@ -114,13 +124,13 @@ def test_flat_label_schema_serialization(self):
                 name=name,
                 domain=Domain.CLASSIFICATION,
                 creation_date=cur_date,
-                id=ID(i),
+                id=ID(str(i)),
```

Comment:
nit: str is not required

Reply:
thanks, rolled back changes in test_flat_label_schema_serialization